mail_notification_custom_subject/models/mail_message_custom_subject.py

```diff
@@ -21,6 +21,9 @@ class MailMessageCustomSubject(models.Model):
         string="Applied Subtypes",
         required=True,
     )
+    subject_to_replace = fields.Char(
+        help="Subject to replace (placeholders may be used here)"
```

Comment:
suggestion: No need to repeat the field name. Instead, it's more helpful to provide placeholder examples. There are lots of ways to add a placeholder, and users can't guess which one is used here:

```suggestion
        help="The text that will be replaced. You can use placeholders. E.g.: {{ object.company_id.name }}"
```